sections/ublock-origin-specific/ads.txt

```diff
@@ -9,3 +9,6 @@ myonlineradio.hu#$#._bannerTop1:style(background-color: transparent !important;)
 myonlineradio.hu##.topHorizontalBanner
 
 /inside-banner.mini.js$domain=myonlineradio.hu
+
+||vezess2.p3k.hu/app/uploads/2023/10/vezess_nyeremenyjatek_alfa-amore_970x250*.png$image
```

Comment:
This rule seems too specific to me, maybe they will use a different image in November